examples/sdk/reactNative/ios-sourcemap-upload.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Script responsible for preprocessing source maps with debugid and uploading it to Backtrace via backtrace-js. 
+# Usage:    ./ios-sourcemap-upload.sh <source_map_file_path> <debug_id_file_path> <backtrace_configuration_path
+# Parameters:
+#   <source_map_file_path>        (Required) Path to the source map file.
+#   <debug_id_file_path>            (Required) path to generated backtrace debug id.
+#   <backtrace_configuration_path>      (Required) Path to the .backtracejsrc configuration file.
+#
+# Adjusting metro configuration is required in order to correctly use debug_id available in the debug_id_file_path.
+
+
+set -e
+set -x
+
+if [ -z "$1" ]; then
+    echo "Error: Missing path to the source map file."
+    exit 1
+fi
+
+source_map_file_path="$1"
+
+# Check if the file exists
+if [ ! -f "$source_map_file_path" ]; then
+    echo "Error: File '$source_map_file_path' does not exist."
+    exit 1
+fi
+
+DEBUG_ID_PATH_ENV="DEBUG_ID_PATH"
+
+debug_id_file_path=${!DEBUG_ID_PATH_ENV:-${2:-}}
```

Comment:
what's the benefit of `DEBUG_ID_PATH_ENV`?

Reply:
I'm assuming something might go wrong in an unknown environment. I still want to give the user and option to override the place where we generate a debugId.

Reply:
how do they override it? They would have to edit the script directly

Reply:
no - they need to set it in the env variable